common/context.go

```diff
@@ -155,10 +182,9 @@ func getRelMuFile(absMuFile string) (string, error) {
 }
 
 // InitializeConfig loads config object
-func (ctx *Context) InitializeConfig(configReader io.Reader) error {
-
+func (ctx *Context) InitializeConfig(yamlContent string) error {
```

Comment:
this should remain as an `io.Reader` to allow the function to be called streams rather than always assuming the entire object is in memory.